syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -423,11 +444,15 @@ def __itruediv__(self, *args, **kwargs):
 
     @overloaded.method
     def __truediv__(self, shares: dict, divisor):
+        # TODO: truediv in Zq
         assert isinstance(divisor, int)
 
         divided_shares = {}
-        for location, pointer in shares.items():
-            divided_shares[location] = pointer / divisor
+        for i_worker, (location, pointer) in enumerate(shares.items()):
+            if i_worker % 3 == 0:
+                divided_shares[location] = self.field - (self.field - pointer) / divisor
```

Comment:
Have checked that type(int / int) == int ?

Reply:
It's not the case with native Python types but `/`does not change the type of torch tensors